modules/ROOT/pages/runtime-patching.adoc

```diff
@@ -30,19 +29,20 @@ By default, MUnit runs with the runtime corresponding to the _minMuleVersion_ of
     </configuration>
 </plugin>
 ----
-<1> Remember that this version is an example. You need to declare the correct version provided by the team.
+<1> This version is a placeholder example. You must declare the correct version provided by MuleSoft.
 
-== Patching MUnit from Studio
+== Patching MUnit Using Studio
 
-By default, MUnit runs in studio with the version of the project’s server. You can override this version, by specifying the patched runtime version in the MUnit run configuration:
+By default, MUnit runs in Anypoint Studio with the Mule runtime engine version of the project’s server. You can override this version by specifying the patched Mule version in the MUnit run configuration menu:
 
 . In the top menu bar, go to *Run* and *Run Configurations…*.
-. Under MUnit, select your MUnit test file.
-. Type the MUnit patched version you want to use in the *Mule Runtime Version* field, under *Advanced Settings*.
+. Under *MUnit*, select your MUnit test file.
+. In the *Mule Runtime Version* field under *Advanced Settings* type the MUnit patched version you want to use.
```

Comment:
```suggestion
. In the *Mule Runtime Version* field under *Advanced Settings*, type the MUnit patched version you want to use.
```

Reply:
Is the MUnit version patched or is the Mule version patched?

Reply:
Good catch. That's supposed to be the Mule version.